submarine-workbench/workbench-web/src/app/pages/workbench/manager/user-drawer/user-drawer.component.ts

```diff
@@ -153,6 +153,8 @@ export class UserDrawerComponent implements OnInit, OnChanges {
 
           if (formData.birthday) {
             formData.birthday = format(formData.birthday, 'yyyy-MM-dd HH:mm:ss');
+          } else {
+            formData.birthday = format(new Date(2000,0O0,0O1),'yyyy-MM-dd HH:mm:ss');
```

Comment:
What time is `Date(2000,0O0,0O1)`?

Reply:
I have updated this file. Now if user do not enter the birthday, the value stored into database will be null.

Reply:
Okay, What we should do is to encrypt the password in the database.